src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/AssessmentStatus.kt

```diff
@@ -89,21 +110,68 @@ enum class AssessmentStatus {
         TaskProgress.Fixed(CREATE_LICENCE, LOCKED),
       ),
     )
+    override val visibleToRole = mapOf(
+      PRISON_CA to true,
+      PROBATION_COM to true,
+      PRISON_DM to false,
+    )
   },
 
-  AWAITING_PRE_DECISION_CHECKS,
+  AWAITING_PRE_DECISION_CHECKS {
+    override val visibleToRole = mapOf(
+      PRISON_CA to true,
+      PROBATION_COM to false,
+      PRISON_DM to false,
+    )
+  },
 
-  AWAITING_DECISION,
+  AWAITING_DECISION {
+    override val visibleToRole = mapOf(
+      PRISON_CA to true,
+      PROBATION_COM to false,
+      PRISON_DM to true,
+    )
+  },
 
-  APPROVED,
+  APPROVED {
+    override val visibleToRole = mapOf(
+      PRISON_CA to true,
+      PROBATION_COM to false,
+      PRISON_DM to true,
+    )
+  },
 
-  AWAITING_PRE_RELEASE_CHECKS,
+  AWAITING_PRE_RELEASE_CHECKS {
+    override val visibleToRole = mapOf(
+      PRISON_CA to true,
+      PROBATION_COM to false,
+      PRISON_DM to false,
+    )
+  },
 
-  PASSED_PRE_RELEASE_CHECKS,
+  PASSED_PRE_RELEASE_CHECKS {
```

Comment:
I know this was my orig suggestion but I think this would probably be slightly more elegant as a set of roles that it's visible to

But this is fine for now! 
Looks good!